app/locales/es/translations.js

```diff
@@ -265,53 +327,78 @@ export default {
     medication: 'Medicación',
     status: 'Estado',
     addNewOutpatientVisit: '--Nueva visita a paciente externo--',
-    prescription: 'Receta medica',
-    prescriptionDate: 'Fecha de la receta medica',
+    prescription: 'Receta médica',
+    prescriptionDate: 'Fecha de la receta médica',
     billTo: 'Facturar a',
     pullFrom: 'Traer de',
     fulfilled: 'Cumplido',
     deleteRequest: 'Eliminar pedido',
-    location: 'Ubicacion',
+    location: 'Ubicación',
     provider: 'Proveedor',
     with: 'con',
-    allDay: 'Todo el dia',
+    allDay: 'Todo el día',
     physician: 'Médico',
     assisting: 'Asistencia',
     anesthesia: 'Anestesia',
     procedures: 'Procedimiento'
```

Comment:
You're missing a comma after `procedures: 'Procedimiento'`.  Change to `procedures: 'Procedimiento',` and it should work!